internal/storage/minio_chunk_manager.go

```diff
@@ -91,30 +91,37 @@ func newMinioChunkManagerWithConfig(ctx context.Context, c *config) (*MinioChunk
 		Client:     minIOClient,
 		bucketName: c.bucketName,
 	}
-	log.Debug("minio chunk manager new minio client success.")
+	log.Info("minio chunk manager new minio client success.")
```

Comment:
necessary to print related configuration here? for example, bucket name, root path and etc.